src/iop/basecurve.c

```diff
@@ -1783,6 +1828,15 @@ static void exstep_callback(GtkWidget *widget, gpointer user_data)
   dt_dev_add_history_item(darktable.develop, self, TRUE);
 }
 
+static void exbias_callback(GtkWidget *widget, gpointer user_data)
```

Comment:
well, we already have exstep_callback, frankly I would rename this too to: exposure_step_callback